pom.xml

```diff
@@ -219,6 +219,8 @@
         <commons-exec.version>1.3</commons-exec.version>
         <commons-collections4.version>4.4</commons-collections4.version>
 
+        <docker-java.version>3.3.3</docker-java.version>
```

Comment:
I think there is no need to declare docker java in pom.xml, it seems that only the graph core module will use it?